core/capabilities/ccip/delegate.go

```diff
@@ -278,17 +278,35 @@ func (d *Delegate) getTransmitterKeys(ctx context.Context, relayIDs []types.Rela
 			return nil, fmt.Errorf("error parsing chain ID, expected big int: %s", relayID.ChainID)
 		}
 
-		ethKeys, err := d.keystore.Eth().EnabledAddressesForChain(ctx, chainID)
-		if err != nil {
-			return nil, fmt.Errorf("error getting enabled addresses for chain: %s %w", chainID.String(), err)
-		}
+		switch relayID.Network {
+		case relay.NetworkEVM:
+			ethKeys, err := d.keystore.Eth().EnabledAddressesForChain(ctx, chainID)
+			if err != nil {
+				return nil, fmt.Errorf("error getting enabled addresses for chain: %s %w", chainID.String(), err)
+			}
 
-		transmitterKeys[relayID] = func() (r []string) {
-			for _, key := range ethKeys {
-				r = append(r, key.Hex())
+			transmitterKeys[relayID] = func() (r []string) {
+				for _, key := range ethKeys {
+					r = append(r, key.Hex())
+				}
+				return
+			}()
+		case relay.NetworkSolana:
```

Comment:
Can we also add `NetworkAptos`, `NetworkCosmos` and `NetworkStarknet` definitions? They'll all look identical to the Solana case and it'll save us time in the future